kubernetes/examples/hello-spring-boot/skaffold.yaml

```diff
@@ -5,8 +5,13 @@ build:
   - image: gcr.io/gcp-dev-tools/hello-spring-boot
     jibMaven: {}
 
-# optional profile to run the jib build on Google Cloud Build
 profiles:
+  # optional profile to run the Jib build on Google Cloud Build
   - name: gcb
     build:
       googleCloudBuild: {}
+  # optional profile to run the build using Docker instead of Jib
```

Comment:
@briandealwis or @chanseokoh 

In this `docker` profile I'm trying to override the Jib builder to just use a local Docker builder. However, doing something like the following:
```
profiles: 
   - name: docker
     build: 
         local: {}
```
Doesn't override the Jib builder. Is there a more elegant way to do this instead of redefining the artifacts / image like I'm doing here?

Reply:
@ivanporty I asked around and haven't heard a good solution so we can merge with this for now (it works).

Reply:
I wonder if you could override the`jibMaven` section and skip the jib step. Something like

```
jibMaven:
  args:
  - -Djib.skip=true
```
(Untested, YMMV, Etc)

Reply:
This is probably a bug of Skaffold?